internal/commands/prepare/prepare.go

```diff
@@ -40,6 +41,9 @@ func NewCommand() *cobra.Command {
 		"Override default image memory configuration (size in megabytes or \"auto\")")
 	command.PersistentFlags().StringArrayVar(&customDirectoryMounts, "dir", []string{},
 		"\"--dir\" arguments to pass to \"tart run\", can be specified multiple times")
+	command.PersistentFlags().BoolVar(&noAutoPrune, "no-auto-prune", false,
```

Comment:
Doesn't it automatically support --no prefix for bool flags? Maybe invert it and default to true to always pool?

Reply:
>Doesn't it automatically support --no prefix for bool flags?

I don't think so: https://github.com/spf13/pflag/issues/214.

Reply:
See https://github.com/cirruslabs/gitlab-tart-executor/pull/42/commits/bebf869f7a9e41ea61bc094a425207977a5890af.